.github/workflows/release-please.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  push:
+    branches:
+      - feature/release-notes-automation
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: release-please
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
```

Comment:
I think one of the things recently suggested by security was pinning actions to very specific versions.  Might as well get in the habit now.  Just a suggestion, I'm not going to hold up approval for it.

Reply:
Yeah that's a good idea, I'll think about that